tap_pipedrive/streams/dealflow.py

```diff
@@ -2,18 +2,20 @@
 from tap_pipedrive.stream import PipedriveIterStream
 
 class DealStageChangeStream(PipedriveIterStream):
-	base_endpoint = 'deals'
-	id_endpoint = 'deals/{}/flow'
-	schema = 'dealflow'
-	state_field = 'log_time'
-	key_properties = ['id', ]
+    base_endpoint = 'deals'
+    id_endpoint = 'deals/{}/flow'
+    schema = 'dealflow'
+    state_field = 'log_time'
+    key_properties = ['id', ]
```

Comment:
I love that we are ripping out the tabs here. Could we also make sure every other file gets converted to spaces as well?

Reply:
All files that I have touched should now be tab free.